app/controllers/users_controller.rb

```diff
@@ -19,10 +28,29 @@ def create
     end
   end
 
+  def update
+    if @user.update user_params
+      flash[:success] = t "user.noti.update_true"
+      redirect_to @user
+    else
+      flash[:danger] = t "user.noti.update_fail"
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t "user.noti.destroy_true"
+    else
+      flash[:danger] = t "user.noti.destroy_fail"
+    end
+    redirect_to users_url
```

Comment:
users_path dc chứ nhỉ

Reply:
Dạ vâng ạ. Em có tìm hiểu thì:

1. path là đường dẫn tương đối trong khi url là đường dẫn tuyệt đối
2. root_url => http://localhost:3000/
    root_path => /
    users_url => http://localhost:3000/users
    users_path  => /users
3. _path thường hay dùng trong các link_to
4. _url thường được dùng để chuyển hướng redirect-to